### PR TITLE
Localization improvements

### PR DESCRIPTION
**Pull request by drzix**
Hi, Anders.

I try to change some codes in FileManager.
I wrote commit messages on all commits.
But I wonder if you don't mind this changes.

Please check my commits in detail

### CODE REVIEW

File: src/client/javascript/locales/en_EN.js [hunk 1]
```diff
@@ -375,7 +375,11 @@
     'LBL_BUGREPORT' : 'Bugreport',
     'LBL_INSTALL' : 'Install',
     'LBL_UPDATE' : 'Update',
-    'LBL_REMOVE' : 'Remove'
+    'LBL_REMOVE' : 'Remove',
+    'LBL_SHOW_SIDEBAR' : 'Show sidebar',
+    'LBL_SHOW_NAVIGATION' : 'Show navigation',
+    'LBL_SHOW_HIDDENFILES' : 'Show hidden files',
+    'LBL_SHOW_FILEEXTENSIONS' : 'Show file extensions'
```
Comment (andersevenrud) [inline src/client/javascript/locales/en_EN.js]: Some of these are also used in https://github.com/os-js/OS.js/blob/master/src/packages/default/Settings/scheme.html#L428 :)

**Comment by andersevenrud** [comment]
Really nice! I added a comment for you to read :)

File: src/client/javascript/locales/en_EN.js (hunk 1 above)
Comment (drzix) [inline src/client/javascript/locales/en_EN.js]: Oops I read this just now. You mean i have to push an additional commit to handle that paragraph?

Comment (andersevenrud) [inline src/client/javascript/locales/en_EN.js]: No, I just hinted that this sentence was also used in the Settings app. Maybe you can add these there as well ?!

**Comment by drzix** [comment]
Well, I did it.
However, strings in Settings/schmes.html have different styles with each other. Something is replaced from **./locale.js** and the other thing replaced from **client/../locales/lang_LANG.js**.
I think it's so complicated for new translators in the future to translate **what to do** and **where to do**.
I want to ask you how to think about this.

**Comment by andersevenrud** [comment]
> I think it's so complicated for new translators in the future to translate what to do and where to do.

Not really (I cannot make everything "beginner safe"). And even if it is I would not worry too much -- this is easily fixable by writing a bit more in-depth translation tutorials.

**Comment by drzix** [comment]
I got what you mean. Then, something I can fix and translation in FileManager seems to be finished, for now.

**Comment by drzix** [comment]
Should I do something more for this PR to be merged??:o

**Comment by andersevenrud** [comment]
Looks good to me! Merging :)

**Pull request merged**